### PR TITLE
Feature/profile

### PR DESCRIPTION
**Pull request by CorneliusMoe**
profile page

pls review :)

### CODE REVIEW

File: profile.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    </main>
+    <footer>
+      <nav class="navbar">
+        <a href="/index.html" class="nav_icons"
+          ><img src="assets/images/home_dark.svg" alt="home"
+        /></a>
+        <a href="/bookmarks.html" class="nav_icons"
+          ><img src="assets/images/bookmark_dark.svg" alt="bookmarks"
+        /></a>
+        <a href="#main" class="nav_icons navActive"
```
Comment (AliaksandraNovikava) [inline profile.html:60]: I don't think you need 'href="#main"' here. You set this attribute on your profile icon, so it doesn't make sense to link it to your main content.

File: profile.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1><a href="/index.html">Quiz App</a></h1>
+    </header>
+    <main id="main">
```
Comment (AliaksandraNovikava) [inline profile.html:14]: I don't think you need 'href="#main"' here. Please read my comment down below =)

File: index.html [hunk 3]
```diff
@@ -11,70 +11,89 @@
...
+      </section>
     </main>
     <footer>
-        <nav class="navbar">
-        <a href="#main" class="nav_icons navActive"><img src="assets/images/home_light.svg" alt="home"></a>
-        <a href="/bookmarks.html" class="nav_icons"><img src="assets/images/bookmark_dark.svg" alt="bookmarks"></a>
-        <a href="profile/profile.html" class="nav_icons"><img src="assets/images/profile_dark.svg" alt="profile"></a>
-        </nav>
+      <nav class="navbar">
+        <a href="#main" class="nav_icons navActive"
```
Comment (AliaksandraNovikava) [inline index.html:87]: I don't think you need 'href="#main"' here. Wouldn't it be better to replace 'main' with '/index.html'? So you don't need to give your main element an id?.

File: index.html [hunk 4]
```diff
@@ -11,70 +11,89 @@
     <header>
       <h1><a href="#">Quiz App</a></h1>
     </header>
-    <main id="main">
+    <main id="main" class="main-flex">
```
Comment (AliaksandraNovikava) [inline index.html:14]: I don't think it's necessary to give your main element an id.

File: bookmarks.html [hunk 5]
```diff
@@ -57,7 +57,7 @@ <h1><a href="/index.html">Quiz App</a></h1>
         <a href="#main" class="nav_icons navActive"
```
Comment (AliaksandraNovikava) [inline bookmarks.html:57]: I'm not sure why you need to link your nav-icon to the main content on the same page.

**Comment by CorneliusMoe** [comment]
Thanks for reviewing Alex, I change my anchors with your comments in mind :)

**Pull request merged**